stable/prometheus/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: prometheus
-version: 8.9.1
+version: 8.9.2
```

I don't think this should be a patch increment.
```suggestion
version: 8.10.0
```

---

Probably should be a major version bump, as it does more then adding a feature too.